src/libs/AxelarQueryAPI.ts

```diff
@@ -240,4 +240,49 @@ export class AxelarQueryAPI {
     result.common_key = assetConfig.common_key[this.environment];
     return result;
   }
+
+  /**
+   * Get a list of active chains.
+   * @returns an array of active chains
+   */
+  public async getActiveChains(): Promise<string[]> {
+    await this.initQueryClientIfNeeded();
+
+    return this.axelarQueryClient.nexus
+      .Chains({ status: ChainStatus.CHAIN_STATUS_ACTIVATED })
+      .then((resp) => resp.chains.map((chain) => chain.toLowerCase()));
```

Comment:
In keeping with the use of exact chain identifiers everywhere, we shouldn't lower case the chain names here

Reply:
Thanks @milapsheth . Fixed  🙏🏻